azure-pipelines.yml

```diff
@@ -99,6 +99,26 @@ stages:
         projects: |
           $(Build.SourcesDirectory)/src/MIDebugEngine-Unix.sln
         arguments: '-c Debug'
+    - script: |
+        sudo apt-get install gdb g++ -y
+        which g++
+        which gdb
+        echo 0 | sudo tee /proc/sys/kernel/yama/ptrace_scope
+        # TODO: Try to make core dumps work
+        # echo 1 | sudo tee /proc/sys/kernel/core_uses_pid
+        # ulimit -S -c unlimited
+        # sudo sysctl -w kernel.core_pattern=$(Build.SourcesDirectory)/core.%e
+    - script: |
+        dotnet publish $(Build.SourcesDirectory)/src/OpenDebugAD7/OpenDebugAD7.csproj -c Debug -r linux-x64 --self-contained
+        mkdir -p $(Build.SourcesDirectory)/bin/DebugAdapterProtocolTests/Debug/extension/debugAdapters
+        cp -r  $(Build.SourcesDirectory)/bin/Debug/vscode/linux-x64/publish/* $(Build.SourcesDirectory)/bin/DebugAdapterProtocolTests/Debug/extension/debugAdapters/.
+        cp $(Build.SourcesDirectory)/bin/Debug/vscode/Microsoft.MIDebugEngine.dll $(Build.SourcesDirectory)/bin/DebugAdapterProtocolTests/Debug/extension/debugAdapters/.
+        cp $(Build.SourcesDirectory)/bin/Debug/vscode/Microsoft.MICore.dll $(Build.SourcesDirectory)/bin/DebugAdapterProtocolTests/Debug/extension/debugAdapters/.
+        cp $(Build.SourcesDirectory)/bin/DebugAdapterProtocolTests/Debug/CppTests/TestConfigurations/config_gdb.xml $(Build.SourcesDirectory)/bin/DebugAdapterProtocolTests/Debug/CppTests/config.xml
+    - task: DotNetCoreCLI@2
+      inputs:
+        command: 'test'
+        arguments: '$(Build.SourcesDirectory)/bin/DebugAdapterProtocolTests/Debug/CppTests/CppTests.dll'
```

Comment:
>arguments [](http://example.com/codeflow?start=8&length=9)

Are you planning to eventually run these on other platforms too?

Reply:
Looking to see if I can get Windows (MinGW) tested on Azure Pipelines. They seem to pull the C++ toolschain via Chocolatey. I was also hoping to spend time to investigate a setup via MSYS2.